bench/commands/setup.py

```diff
@@ -358,7 +358,7 @@ def sync_domains(domain=None, site=None):
 @click.command("role", help="Install dependencies via ansible roles")
 @click.argument("role")
 @click.option("--admin_emails", default="")
-@click.option("--mysql_root_password")
+@click.option("--mariadb_root_password")
```

Comment:
We shouldn't change this, this can be breaking change for some scripts. 

Add another alias for same flag? IIRC `click` does support that.